README.rst

```diff
@@ -360,40 +398,57 @@ In short, the more ``-v`` arguments you supply, the more verbose the output gets
-file references passed to ``aws-encryption-cli`` are identified by the ``@`` prefix and the
-contents are expanded as if you had included them in line. Configuration files can have any
-name you desire.
+.. warning::
+
+   There is a `known issue with configuration file parsing in Windows`_. On Windows
+   only, configuration files cannot contain quotation marks (single or double).
+   ``aws-encryption-cli`` commands fail if they refer to configuration files that
+   contain quotation marks. If this affects you, please let us know by filing an issue
+   in our `GitHub`_ repo.
+
+
+Configuration files are supported by using Python's native `argparse file support`_,
+which allows you to write configuration files exactly as you would enter arguments in
+the shell.
+
+Configuration files are text files. They can have any valid file name and extension.
+
+To refer to a configuration file in an ``aws-encryption-cli`` command, prefix the file
+name with ``@``.
 
 .. note::
 
-   In PowerShell, you will need to escape the ``@`` symbol so that it is sent to ``aws-encryption-cli``
-   rather than interpreted by PowerShell.
+   In PowerShell, use a backtick to escape the ``@`` symbol (```@``) so that the
+   configuration file reference is sent to ``aws-encryption-cli`` and not interpreted
+   by PowerShell.
 
-For example, if I wanted to use a common master key configuration for all of my calls, I could
-create a file ``master-key.conf`` with contents detailing my master key configuration.
+For example, to use a common master key configuration for multiple commands, create
+a ``master-key.conf`` file that contains the parameters and parameter values that
+describe your master key configuration.
 
 **master-key.conf**
 
 .. code-block:: sh
 
    --master-key key=A_KEY key=ANOTHER_KEY
 
-Then, when calling ``aws-encryption-cli``, I can specify the rest of my arguments and reference
-my new configuration file, and ``aws-encryption-cli`` will use the composite configuration.
+In the ``aws-encryption-cli`` command, enter the remaining parameters and reference
+the configuration file. The ``aws-encryption-cli`` combines the parameters on the
+command line with the parameters in the configuration file.
 
 .. code-block:: sh
 
-   aws-encryption-cli -e -i $INPUT_FILE -o $OUTPUT_FILE @master-key.conf
-
+   aws-encryption-cli -e -i $INPUT_FILE -o $OUTPUT_FILE --metadata-output $METADATA_FILE @master-key.conf
 
-To extend the example, if I wanted a common caching configuration for all of my calls, I could
-similarly place my caching configuration in a configuration file ``caching.conf`` in this example
-and include both files in my call.
+   To create a configuration file that saves your data key caching settings, save
+   the caching parameter and its attributes in a configuration file. The following
+   example creates a ``caching.conf`` configuration file and uses it in two different
+   commands.
```

Comment:
Un-indent these. Having them indented will pull them into the code-block directive, and that is not what we want.